modules/gathering-data-specific-features.adoc

```diff
@@ -78,7 +78,7 @@ ifndef::openshift-rosa,openshift-dedicated[]
 |`registry.redhat.io/lvms4/lvms-must-gather-rhel9:v<installed_version_LVMS>`
 |Data collection for the LVM Operator.
 
-|`ghcr.io/complianceascode/must-gather-ocp`
+|`registry.redhat.io/compliance/openshift-compliance-must-gather-rhel8`
```

Comment:
Seems the latest tag was not available.
```
% podman pull registry.redhat.io/compliance/openshift-compliance-must-gather-rhel8
Trying to pull registry.redhat.io/compliance/openshift-compliance-must-gather-rhel8:latest...
Error: initializing source docker://registry.redhat.io/compliance/openshift-compliance-must-gather-rhel8:latest: reading manifest latest in registry.redhat.io/compliance/openshift-compliance-must-gather-rhel8: unsupported: This repository does not use the "latest" tag to track the most recent image and must be pulled with an explicit version or image reference. For more information, see: https://access.redhat.com/articles/4301321
```

Reply:
The digest or detailed version works. The latest tag doesn't work.
% podman pull registry.redhat.io/compliance/openshift-compliance-must-gather-rhel8:1.6.1-2 
Trying to pull registry.redhat.io/compliance/openshift-compliance-must-gather-rhel8:1.6.1-2...
Getting image source signatures
Checking if image destination supports signatures
Copying blob 36a6aa53b1de skipped: already exists  
Copying blob d3c06847d186 skipped: already exists  
Copying config 3e4648d4fe done  | 
Writing manifest to image destination
Storing signatures
3e4648d4fe6a794e19921eaf592868d325586f4a1003dd6e25a1e4955cb4eaae

Reply:
Hi @xiaojiey I must have missed this update, sorry about that. 

Should the latest tag work? Is this an engineering issue? 

Or do we need to change the text to `...must-gather-rhel8:<detailed-version>`?

Reply:
I think latest tag should work. I created https://issues.redhat.com/browse/OCPBUGS-50856 to track.
Maybe we can change it to ...must-gather-rhel8:<detailed-version>.
The probem is: it is hard to get detailed version for customers.
can we change it to registry.redhat.io/compliance/openshift-compliance-must-gather-rhel8:1.6.1-2 and remove the detailed version once the related bug OCPBUGS-50856  got fixed?
Or shall we wait until OCPBUGS-50856  got fixed?
@rhmdnd Any comment? Thanks.

Reply:
I believe the repository requires digest or specific versions to pull the images.

Can we update the direction to use that approach instead?

Reply:
I updated the PR to use the digest version "rhel8:1.6.1-2". Is this the direction want to go?

Reply:
We might also want to help users find that image, since it will change with each release.

Reply:
@rhmdnd i've searched the docs but I can't find anything to help users find the image. Any ideas?

We could also do some sort of variable like "...rhel8:<digest-version>" as a lot of the table is [referenced in similar ways](https://87063--ocpdocs-pr.netlify.app/openshift-enterprise/latest/support/gathering-cluster-data.html#gathering-data-specific-features_gathering-cluster-data).

Reply:
They'd probably have to search the CVS and pull out the related images for the must-gather image.

We did something similar upstream https://github.com/ComplianceAsCode/compliance-operator/blob/master/doc/usage.md#must-gather-support

I'm also fine using `<digest-version>`.